### PR TITLE
Bitcoin integration revisions 

### PR DESCRIPTION
**Pull request by jessiemongeon1**
Thank you for your contribution to the IC Developer Portal.
Before submitting your Pull Request, please make sure that:

- [ ] You have updated the [`.github/CODEOWNERS`](https://github.com/dfinity/portal/blob/master/.github/CODEOWNERS) file according to your changes.
- [ ] You have registered new documents to [`/sidebars.js`](https://github.com/dfinity/portal/blob/master/sidebars.js) if any.

### CODE REVIEW

File: docs/developer-docs/integrations/bitcoin/submit-transactions.mdx [hunk 1]
```diff
@@ -0,0 +1,85 @@
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Sending Bitcoin transactions 
+
+## Overview
+
+To submit transactions to the Bitcoin network, the Bitcoin integration API exposes the following method:
```
Comment (ulan) [inline docs/developer-docs/integrations/bitcoin/submit-transactions.mdx:8]: The canister has to first sign the transaction before sending it to Bitcoin. Signing happens with threshold ECDSA. Wdyt about inlining some of the threshold ECDSA code snippets here as the first step?

File: docs/developer-docs/integrations/bitcoin/read-state.mdx [hunk 2]
```diff
@@ -0,0 +1,215 @@
...
+
+Then make a call to this canister's function with the command:
+
+```
+dfx canister --network=ic call CANISTER_NAME get_balance '("ADDRESS")'
+```
+
+Replace `CANISTER_NAME` with your canister's name and replace `ADDRESS` with your Bitcoin address. [Learn more about Bitcoin addresses.](https://en.bitcoin.it/wiki/Transaction#Pay-to-PubkeyHash)
+
+## Reading the fee percentile
```
[the remark this answers is not in the document]
Comment (ulan) [reply docs/developer-docs/integrations/bitcoin/read-state.mdx:152]: I think we could put the following description here:

The transaction fees in the Bitcoin network change dynamically based on the number of pending transactions. In order to get fee percentiles for the last few blocks (4 - 10), call `bitcoin_get_current_fee_percentiles`.

The function returns 101 numbers that are fees measured in millisatoshi per virtual byte. The `i`-th element of the result corresponds to the `i`-th percentile fee. For example, to get the median fee over the last few blocks, look at the `50`-th element of the result.

**Review by ulan — COMMENTED** [review]
Thanks! Looking good overall.

**Review by ulan — APPROVED** [review]

File: docs/developer-docs/integrations/bitcoin/submit-transactions.mdx [hunk 3]
```diff
@@ -0,0 +1,180 @@
...
+
+- `bitcoin_send_transaction`: Submits a transaction to the Bitcoin network.
+
+
+## Signing transactions 
+
+Before a transaction can be sent to the Bitcoin network, it must be signed using Threshold ECDSA API method `sign_with_ecdsa` and an ECDSA public key, which can be returned by calling the API method `ecdsa_public_key`.
+
+An code example using these API methods can be found below:
```
Comment (dsarlis) [inline docs/developer-docs/integrations/bitcoin/submit-transactions.mdx:17]:
```suggestion
A code example using these API methods can be found below:
```

**Review by dsarlis — COMMENTED** [review]
LGTM. Left some editorial comments only.

**Review by dsarlis — APPROVED** [review]

**Review by dfx-json — APPROVED** [review]

**Pull request merged**